src/css/controls/imports/settings-menu.less

```diff
@@ -0,0 +1,61 @@
+@import "../../shared-imports/vars";
+
+.jw-settings-menu {
+    display: none;
+    position: absolute;
+    z-index: -1;
+    flex-flow: column wrap;
+    align-items: flex-start;
+    bottom: @controlbar-height;
+    right: 0;
+    height: ~"calc(100% - @{controlbar-height})";
```

Comment:
By setting `height`, `width` and `right` to these values, we'd be optimizing for the small-player breakpoints. Does the most common player size fall within breakpoint 0-2?

Alternatively, if you make the defaults the values for breakpoints 5-7, you'd reduce the css output.